services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:

```

Comment:
The `tickets` parameter is a list of dictionaries. It's better to use a more descriptive name for it, for example, `ticket_data_list`. It will make your code more readable.